app/client/components/svgs/roundel.tsx

```diff
@@ -1,15 +1,31 @@
 import React from "react";
+import { minWidth } from "../../styles/breakpoints";
+import { gridColumns, gridItemPlacement } from "../../styles/grid";
 
 export interface RoundelProps {
-  size: number;
+  size?: number;
   fillMain?: string;
   fillG?: string;
 }
 
 export const Roundel = (props: RoundelProps) => (
   <a
     css={{
-      alignSelf: "flex-start"
+      ...gridItemPlacement(-2, 1),
```

Comment:
Not so sure about all the _placement_ code residing in the component. Appreciate there probably aren't any non-grid uses of the roundel currently but might be worth at least wrapping it in another component for placement like `GridRoundel` if you want to avoid repetition of the placement code